### PR TITLE
Add Descriptions.

### PR DESCRIPTION
**Pull request by aranajhonny**
Great Work, Justin. This PR is for adding the missing descriptions.

### CODE REVIEW

File: memconfig.json [hunk 1]
```diff
@@ -481,95 +576,115 @@
...
                 "name": "page",
                 "type": "Int",
-                "optional": true
+                "optional": true,
+                "description": "Page number"
               },
               {
                 "name": "pageSize",
                 "type": "Int",
-                "ofType": "Int",
```
Comment (aranajhonny) [inline memconfig.json:489]: This is not necessary

File: memconfig.json [hunk 2]
```diff
@@ -481,95 +576,115 @@
...
+            "description": "The user id that posted that activity"
           },
           {
             "name": "type",
-            "type": "String"
+            "type": "String",
+            "description": "Type of the activity"
           },
           {
-            "name": "messasge",
```
Comment (aranajhonny) [inline memconfig.json:524]: Typo

File: memconfig.lock [hunk 3]
```diff
@@ -21,19 +21,19 @@
                 "params": [
                   {
                     "name": "url",
-                    "description": "The URL to send the POST request to",
```
Comment (juancampa) [inline memconfig.lock]: @aranajhonny when you get a chance, update your `http` and I think most of these changes will go away.

File: memconfig.lock [hunk 4]
```diff
@@ -284,20 +284,6 @@
...
-                "params": [
-                  {
-                    "name": "url",
-                    "description": "",
-                    "type": "String",
-                    "optional": false
-                  }
-                ],
-                "hints": {}
-              },
```
Comment (juancampa) [inline memconfig.lock:300]: For context, this is an experiment Justin and I were working on to make an HTTP node that is scoped to a specific URL, but it's not in http yet, so your removal of it is correct.

**Review by juancampa — CHANGES_REQUESTED** [review]

**Pull request merged**